conda-forge.yml

```diff
@@ -1,2 +1,4 @@
 conda_forge_output_validation: true
 provider: {linux_ppc64le: default, linux_aarch64: default}
+build_platform:
+  osx_arm64: osx_64
```

Comment:
```suggestion
  osx_arm64: osx_64
test_on_native_only: true
```

Reply:
might need a rerender

Reply:
I'm generally not a big fan of `test_on_native_only: true` because we might add some tests in meta.yaml that could be run on cross-compilation (e.g., checking for the presence of some files). But I don't feel strongly about this, so we could do the changes you indicated if you prefer.